samples/kotlin_compose/src/main/res/values/themes.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-    <style name="Theme.CustomerioSDK" parent="android:Theme.Material.Light.NoActionBar" />
+    <style name="Theme.CustomerioSDK" parent="Theme.Material3.Light.NoActionBar" />
```

Comment:
needed because we are going to be showing snackbar in activity and it needs theme.